ddl/db_integration_test.go

```diff
@@ -2510,3 +2510,15 @@ func (s *testIntegrationSuite3) TestIssue20490(c *C) {
 
 	tk.MustQuery("select b from issue20490 order by a;").Check(testkit.Rows("1", "1", "<nil>"))
 }
+
+// TestDefaultValueIsLatin1 for issue #18977
+func (s *testIntegrationSuite3) TestDefaultValueIsLatin1(c *C) {
```

Comment:
I think this bug does not affect only charset latin1 but many other charset. e.g. `utf8mb4`, can you please test it?